enterprise/cypher/cypher/src/main/scala/org/neo4j/cypher/internal/spi/v3_4/codegen/GeneratedMethodStructure.scala

```diff
@@ -61,7 +62,7 @@ object GeneratedMethodStructure {
 }
 
 
-class GeneratedMethodStructure(val fields: Fields, val generator: CodeBlock, aux: AuxGenerator, tracing: Boolean = true,
+class GeneratedMethodStructure(val fields: Fields, val generator: CodeBlock, util: AuxGenerator, tracing: Boolean = true,
```

Comment:
This variable name didn't really need to change

Reply:
...so windows doesn't complain about variables named `aux` you think? :)